### ISSUE TITLE
Add License

### ISSUE THREAD

**Issue by microhobby** [toradex/vscode-torizon-templates#21]
We need be clear about the license of the repo and the code shared.
Cc: @andreriesco @leograba

**Comment by leograba** [comment]
It makes sense, do you have any suggestions @microhobby? Even if not necessarily about a specific license, but about how permissive you think this repo need/should be?

**Comment by microhobby** [comment]
@leograba For being developed and maintained by Toradex and receiving external contributions I think it would be good to be permissive and have compatibility with other licenses, as we use other parts and template code from other frameworks. And I think that the MIT license has the properties that we are seeking. Let me know what you think.

**Comment by leograba** [comment]
It makes sense to me. Users are expected to freely change recently created templates, I don't see a big reason to have a license that is more restrictive.

But I don't think it's our decision to make, let me tag @lucas-tdx and @renato-tdx, who can help us.

**Comment by renato-tdx** [comment]
I agree with not being restrictive. I think it is nice to state that modifications must be licensed under the same licensing, as in other licensing models. However, I did not find anyone that is adequate to our vision here, so MIT works fine for me too.

**Comment by microhobby** [comment]
Ok, so I think we are aligned about it. I will create a PR to add the MIT license on the repo. Thanks!

**Comment by devdotnetorg** [comment]
For the progress of the community, the best solution is the MIT license. Users will be able to independently develop and modify any templates. At least I publish my templates for the [vscode-extension-dotnet-fastiot extension](https://github.com/devdotnetorg/vscode-extension-dotnet-fastiot/tree/dev/templates/system) under the MIT license.

**Comment by microhobby** [comment]
Added in 4b78dcf
